test/primitiveColumns.hs

```diff
@@ -0,0 +1,45 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
Name of the file should be `PrimitiveColumns`.

Reply:
Also, I think we should call it something else. It is not really related to columns, but primitive values of GraphQL.